css/style.css

```diff
@@ -0,0 +1,55 @@
+body{
+    width:100%;
+    font-family:'Segoe UI';
+
+    display: grid;
```

Comment:
- [ ]  I suggest creating a separate class that make use of this display:grid property so you can reuse it without repeating this.

Like this
`.grid{
  
  display:grid;

}
`

You can apply this as needed in the container as the class.